defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# defaults file for sanity-check
+
+SANITY_CHECK_DISK_SPACE:
+    # Mount point
+  - mount: /
+    # Treshold in mb
+    treshold_mb: 5000
```

Comment:
There is a new setting: `SANITY_CHECK_DISK_SPACE_PERCENTAGE` that works for all the fses under `/dev`, and I deleted `SANITY_CHECK_DISK_SPACE`. It might make sense to require that all fs have at least 20% free space at all times --- when I used mb's I needed to set per-partition limits, but in percents I could use global.